tests/Command/RunCommandTest.php

```diff
@@ -313,39 +314,38 @@ public function testExecuteSuccessWithLocalInputOutputInResult(): void
         $job = $client->createJob($job);
 
         putenv('JOB_ID=' . $job->getId());
-        $kernel = static::createKernel();
-        $application = new Application($kernel);
-
-        $command = $application->find('app:run');
-
-        $property = new ReflectionProperty($command, 'logger');
-        $property->setAccessible(true);
-        /** @var Logger $logger */
-        $logger = $property->getValue($command);
-        $testHandler = new TestHandler();
-        $logger->pushHandler($testHandler);
 
-        $property = new ReflectionProperty($command, 'storageClientFactory');
-        $property->setAccessible(true);
-        /** @var StorageClientPlainFactory $baseFactory */
-        $baseFactory = $property->getValue($command);
-        $baseOptions = $baseFactory->getClientOptionsReadOnly();
+        $storageClientFactory = static::getContainer()->get(StorageClientPlainFactory::class);
+        $baseOptions = $storageClientFactory->getClientOptionsReadOnly();
 
         $storageClientFactoryMock = $this->getMockBuilder(StorageClientPlainFactory::class)
             ->setConstructorArgs([$baseOptions])
             ->getMock();
         $storageClientFactoryMock
             ->expects(self::exactly(2))
             ->method('createClientWrapper')
-            ->willReturnCallback(function (ClientOptions $options) use ($baseFactory): ClientWrapper {
+            ->willReturnCallback(function (ClientOptions $options) use ($storageClientFactory): ClientWrapper {
                 $backendConfiguration = $options->getBackendConfiguration();
                 self::assertNotNull($backendConfiguration);
                 self::assertSame('{"context":"123_transformation"}', $backendConfiguration->toJson());
-                return $baseFactory->createClientWrapper($options);
+                return $storageClientFactory->createClientWrapper($options);
             })
         ;
 
-        $property->setValue($command, $storageClientFactoryMock);
```

Comment:
Tohle jsme si trochu sam nadelal do bot - `setValue` nejde provyst, protoze jsem samozrejme proaktivne predelal properties v te class na `readonly`. Nicmene aspon se nam to sjednocuje s tim, jak podobny opicarny delame jinde - ten mock natlacime misto puvodni class primo do kontejneru.

Reply:
Jasně, proč ne, bylo to mizerný předtím, teď je to jenom mizerný jinak :D